src/main/scala/org/broadinstitute/dsde/workbench/avram/dataaccess/HttpSamDao.scala

```diff
@@ -0,0 +1,34 @@
+package org.broadinstitute.dsde.workbench.avram.dataaccess
+
+import java.util.logging.Logger
+
+import com.softwaremill.sttp._
+import io.circe.Decoder.Result
+import io.circe.{DecodingFailure, Json, ParsingFailure}
+import io.circe.generic.auto._
+import io.circe.parser._
+import org.broadinstitute.dsde.workbench.avram.util.{ErrorResponse, RestClient}
+
+class HttpSamDao extends SamDao with RestClient {
+  private val log: Logger = Logger.getLogger(getClass.getName)
+  // TODO: read from config
+  val samUrl = "https://sam.dsde-dev.broadinstitute.org"
+
+  override def getUserStatus(token: String): Either[ErrorResponse, UserInfo] = {
+    val request = buildAuthenticatedGetRequest(samUrl, "/register/user/v1?userDetailsOnly=true", token)
```

Comment:
please use the v2 end point https://sam.dsde-dev.broadinstitute.org/#!/Users/getUserStatusInfo

Reply:
The v2 endpoint does not have a `userDetailsOnly` parameter. Is that a problem or is the v2 endpoint inherently more efficient?